src/session.cpp

```diff
@@ -81,7 +81,13 @@ std::size_t olm::Session::new_outbound_session(
     received_message = false;
     alice_identity_key = alice_identity_key_pair.public_key;
     alice_base_key = base_key.public_key;
-    bob_one_time_key = one_time_key;
```

Comment:
This is an instance variable, so I checked everywhere it's referenced to see if there would be any issues to changed it. It seems safe